### PR TITLE
PrestaShop 9 Compability + Namespace fix

### PR DESCRIPTION
**Pull request by 0x346e3730**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fixes the module's Namespace + compability for PS9
| Type?         | bug fix
| BC breaks?    | yes
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Install the module and try to access the "configure" page

### CODE REVIEW

File: config/services.yml [hunk 1]
```diff
@@ -14,44 +14,45 @@ services:
...
-      - 'PrestaShop\KeycloakConnectorDemo\Form\ConfigurationType'
+      - '@PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationDataProvider'
+      - 'PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationType'
       - 'KeycloakConfiguration'
 
-  PrestaShop\KeycloakConnectorDemo\Controller\ConfigurationController:
-    class: PrestaShop\KeycloakConnectorDemo\Controller\ConfigurationController
+  PrestaShop\Module\KeycloakConnectorDemo\Controller\ConfigurationController:
+    public: true
+    autowire: true
```
Comment (0x346e3730) [inline config/services.yml:48]: This right here is the compatibility for PS9, everything else is namespace fix

**Review by nicosomb — APPROVED** [review]

File: tests/phpstan.sh [hunk 2]
```diff
@@ -26,6 +26,6 @@ echo "Run PHPStan using phpstan-${PS_VERSION}.neon file"
 docker run --rm --volumes-from temp-ps \
        -v $PWD:/var/www/html/modules/keycloak_connector_demo \
        -e _PS_ROOT_DIR_=/var/www/html \
-       --workdir=/var/www/html/modules/keycloak_connector_demo ghcr.io/phpstan/phpstan:1.9.8 \
+       --workdir=/var/www/html/modules/keycloak_connector_demo ghcr.io/phpstan/phpstan:1-php8.1 \
```
Comment (0x346e3730) [inline tests/phpstan.sh:29]: Also phpstan needed to be upgraded to use php 8.1

File: config/services.yml (hunk 1 above)
Comment (matks) [reply config/services.yml:48]: Would this mean that _every module developer has to put `autowire: true`  on his symfony module controllers_, else they don't work?

If yes
- we need to understand why (to make sure this is not a symptom of a bigger problem)
- we need to document it

**Review by matks — APPROVED** [review]

**Comment by nicosomb** [comment]
I tried locally and it seems ok for me.

**Review by mflasquin — APPROVED** [review]

**Comment by mflasquin** [comment]
Validated by me and @nicosomb

**Pull request merged**